[GLA-1481] feat: new custom vocab parameter and no limit anymore (#46)
gladiaio/docs: chapters/live-stt/features.mdx

## Custom vocabulary

To enhance the precision of words you know will recur often in your transcription, use the `custom_vocabulary` feature.<br />

```json
{
  "realtime_processing": {
    "custom_vocabulary": true,
    "custom_vocabulary_config": {
      "vocabulary": [
        "Westeros", 

>         { "value": "Stark"}, 

Reviewer: ```suggestion
        {"value": "Stark"}, 
```
or
```suggestion
        { 
          "value": "Stark"
        }, 
```

        { 
          "value": "Night's Watch",
          "pronunciations": ["Nightz Watch"],
          "intensity": 0.4,
          "language": "en"
        }
      ],
      "default_intensity": 0.6
    }
  }
}
```

> Use only a string in place of an object is equivalent to use `{ "value": "string" }`

Reviewer: ```suggestion
Using a string in place of an object is equivalent to use `{ "value": "string" }`
```

- `default_intensity`: [optional] The global intensity of the feature (minimum 0, maximum 1, default 0.5).
- `vocabulary.value`: [required] The text used to replace in the transcription.
- `vocabulary.pronunciations`: [optional] The pronunciations used in the transcription language, or `vocabulary.language`if present. 
- `vocabulary.intensity`: [optional] The intensity of the feature for this particular word (minimum 0, maximum 1, default 0.5).

> - `vocabulary.language`: [optional] Specified the language in witch it will be pronounced when sound comparison occur, use the transcription language else.

Reviewer: ```suggestion
- `vocabulary.language`: [optional] Specify the language in which it will be pronounced when sound comparison occurs. Default to transcription language.
```
